### PR TITLE
Add current package to appium bindings

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

**Review by imurchie — APPROVED** [review]

File: docs/en/writing-running-appium/appium-bindings.md [hunk 1]
```diff
@@ -759,6 +759,46 @@ NSError *err;
...
+```
+
+```python
+# python
+driver.current_package
+```
+
+```java
+// java
+driver.currentPackage();
```
Comment (SrinivasanTarget) [inline docs/en/writing-running-appium/appium-bindings.md]: It should be `driver.getCurrentPackage()`.

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: docs/en/writing-running-appium/appium-bindings.md (hunk 1 above)
Comment (dpgraham) [inline docs/en/writing-running-appium/appium-bindings.md]: Thanks. I updated it now.

File: docs/en/writing-running-appium/appium-bindings.md [hunk 2]
```diff
@@ -785,20 +785,14 @@ driver.getCurrentPackage().then(function (package) { /*...*/ })
 
 ```php
 // php
-$this->currentPackage();
+$this->getCurrentPackage();
```
Comment (SrinivasanTarget) [inline docs/en/writing-running-appium/appium-bindings.md]: @dpgraham I'm not sure about php client.

Comment (SrinivasanTarget) [inline docs/en/writing-running-appium/appium-bindings.md]: Should be for java client.

Comment (dpgraham) [inline docs/en/writing-running-appium/appium-bindings.md]: Good eye. Sorry I rushed that one. I'll fix it again.

**Review by SrinivasanTarget — APPROVED** [review]

File: docs/en/writing-running-appium/appium-bindings.md (hunk 2 above)
Comment (dpgraham) [inline docs/en/writing-running-appium/appium-bindings.md]: Fixed it again.

Comment (SrinivasanTarget) [inline docs/en/writing-running-appium/appium-bindings.md]: Approved 👍

**Review by jlipps — COMMENTED** [review]
👍

**Pull request merged**